templates/zerver/help/mute-topic.md

```diff
@@ -0,0 +1,66 @@
+# Mute a topic
+
+If you would like to stop receiving notifications from a certain topic, you can
+choose to mute it.
+
+There are two main ways of muting a topic; you can mute a topic through the
+stream list or through your view.
+
+## Muting a topic through the stream list
+
+1. Find the stream list located at the left sidebar.
+
+    ![Left sidebar](/static/images/help/left-sidebar.png)
+
+2. In the stream list, click on the stream that the topic you want to mute
+belongs to. Upon clicking on the stream, a dropdown with all the topics in the
+stream will appear.
+
+    ![Stream topic dropdown](/static/images/help/mute-stream-1.png)
+
+3. In the stream list dropdown, hover over the topic to reveal a chevron
+(![chevron](/static/images/help/chevron.png)) icon to the right of the topic.
+
+    ![Stream topic dropdown chevron hover](/static/images/help/topic-chevron-hover.png)
+
+4. Upon clicking on the topic's chevron
+(![chevron](/static/images/help/chevron.png)) icon, a dropdown with topic
+actions will appear. To mute the topic, simply select the **Mute the topic (topic name)**
+option from the dropdown.
+
+    ![Stream topic dropdown mute topic](/static/images/help/mute-topic.png)
+
+5. Upon clicking **Mute the topic (topic name)**, a tooltip titled **Topic Muted** will
+appear, and the topic will be grayed out in the stream list dropdown, confirming
+the success of your muting of the topic.
+
+    ![Topic mute success](/static/images/help/topic-muted.png)
+
+## Muting a topic through your view
+
+1. The default **Home** view displays all messages from unmuted streams and topics
+that you've subscribed to. You can choose to search for messages belonging to
+the topic that you wish to mute in the **Home** view or narrow your view to
+messages sent to the topic that you want to mute by entering the search operator
+`topic:(topic name)` in the search bar.
```

Comment:
The last sentence is a little long in my opinion, you might want to break it down into two phrases or use two bullet points instead of the 'or'.